gcr/ansible-vault.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Ensures ansible-vault is installed within the build environment
+#
+# NOTE: This causes an error when running in Travis
+#       DO NOT USE 'set -e' in this script, otherwise all hell breaks loose,
+#       but, yaknow, without outputing the actual issue to the Travis log...
+
+# Set TOOL_ROOT, the location of the directory this script is housed in
+readonly TOOL_ROOT=$(cd $( dirname "${BASH_SOURCE[0]}" ) && pwd )
+
+# Include shell helpers
+source $TOOL_ROOT/utils.sh
+
+echo_yellow "Ensuring ansible-vault is installed..."
+
+IS_INSTALLED=$(which ansible-vault)
```

Comment:
`which` isn't a a greay way to [tell if something is installed](https://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script). Let's use `hash`, like we do in [`setup-local-machine`](https://github.com/soulcycle/podium-local/blob/master/bin/setup-local-machine#L41)

Reply:
I didn't know that about which. that's great! ty!